packages/webstudio-sdk/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@webstudio-is/sdk",
+  "version": "0.5.0",
+  "description": "Webstudio JavaScript / TypeScript API",
+  "author": "Webstudio <github@webstudio.is>",
+  "repository": "git@github.com:webstudio-is/webstudio-sdk.git",
+  "homepage": "https://webstudio.is",
+  "scripts": {
+    "build": "yarn build:mdn-data && yarn build:args && yarn build:prisma && yarn build:lib",
```

Comment:
we don't need to call many of those every time actually, it was already slowing things down because we called it a lot, e.g.
- build:mdn-data only needs to be built when we need something new from the mdn-data package, which will only happen if we update the dependency
- build:args - will be needed when ./src/components/** changes, but I am not sure how we can prevent doing it too many times
- build:prisma - only when schema.prisma changes, think we need to only do it on postinstall automatically and otherwise call manually, when we change schema, it doesn't change often